test/unit/route-reading.js

```diff
@@ -37,14 +37,14 @@ describe('Reading from $route service', function() {
   
   describe('after setting readFromRoute', function() {
     beforeEach(module(function(AnalyticsProvider, $provide){
-      AnalyticsProvider.readFromRoute(true);
+      AnalyticsProvider.readFromRoute(true)
+              // Use classic analytics because have no idea how to track events for the universal one :D
```

Comment:
This comment is unnecessary. trackPage works for both classic and universal analytics. For universal analytics you use the Analytics.log to see what is being emitted.